docs/install.md

```diff
@@ -5,62 +5,58 @@ sidebar_label: Installation
 
 # Installation
 
-Several options for installing `zq` and/or `zed` are available:
-* [HomeBrew](#homebrew) for Mac or Linux,
-* [Binary Download](#binary-download), or
-* [Build from Source](#building-from-source).
+Several options for installing `super` are available:
+* [Homebrew](#homebrew) for Mac or Linux,
+* [Binary download](#binary-download), or
+* [Build from source](#building-from-source).
 
-To install the Zed Python client, see the
+To install the SuperDB Python client, see the
 [Python library documentation](libraries/python.md).
 
 ## Homebrew
 
-On macOS and Linux, you can use [Homebrew](https://brew.sh/) to install `zq`:
+On macOS and Linux, you can use [Homebrew](https://brew.sh/) to install `super`:
 
 ```bash
-brew install brimdata/tap/zq
-```
-
-Similarly, to install `zed` for working with Zed lakes:
-```bash
-brew install brimdata/tap/zed
+brew install brimdata/tap/super
 ```
 
 Once installed, run a [quick test](#quick-tests).
 
 ## Binary Download
 
-We offer pre-built binaries for macOS, Windows and Linux for both x86 and arm
-architectures in the Zed [Github Release page](https://github.com/brimdata/super/releases).
-
-Each archive includes the build for `zq` and `zed`.
+We offer pre-built binaries for macOS, Windows and Linux for both amd64/x86 and arm
```

Comment:
I figured we might as well mention "amd64" explicitly since that's actually the architecture name that appears in the download artifacts on GitHub.